test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresUnreferencedCodeCapability.cs

```diff
@@ -882,6 +897,87 @@ public class NestedClass
 				}
 			}
 
+			[RequiresUnreferencedCode("Message for --StaticCtor--")]
+			class StaticCtor
+			{
+				static StaticCtor ()
+				{
+				}
+			}
+
+			[ExpectedWarning ("IL2026", "RequiresOnClass.StaticCtor.StaticCtor()", "Message for --StaticCtor--", GlobalAnalysisOnly = true)]
+			[ExpectedWarning ("IL2026", "RequiresOnClass.StaticCtor..cctor()", "Message for --StaticCtor--", GlobalAnalysisOnly = true)]
```

Comment:
I assume the first one should say `GlobalAnalysisOnly=false` - but it's probably not enough, since it will only show up in the analyzer, right?

Reply:
Creating the new instance of the class is calling the .cctor and then .ctor, why would linker show only one warning?